CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_InfiltrationActions.uc

```diff
@@ -32,12 +164,12 @@ static function X2CovertInfiltrationTemplate CreateInfiltrationTemplate(name Cov
 
 	if (bCreateSlots)
 	{
-		Template.Slots.AddItem(CreateDefaultOptionalSlot('InfiltrationStaffSlot'));
 		Template.Slots.AddItem(CreateDefaultStaffSlot('InfiltrationStaffSlot'));
 		Template.Slots.AddItem(CreateDefaultStaffSlot('InfiltrationStaffSlot'));
 		Template.Slots.AddItem(CreateDefaultStaffSlot('InfiltrationStaffSlot'));
 		Template.Slots.AddItem(CreateDefaultStaffSlot('InfiltrationStaffSlot'));
 		Template.Slots.AddItem(CreateDefaultOptionalSlot('InfiltrationStaffSlot'));
+		Template.Slots.AddItem(CreateDefaultOptionalSlot('InfiltrationStaffSlot'));
```

Comment:
What is this doing here?

Reply:
I tried to move the 5th optional to the 1st slot before in this branch, then you told me it wouldn't work so I moved it back.

Reply:
I'm just a bit confused as to how it would result during the merge, since the change is already done on master